### PR TITLE
Clarify MQTT camera documentation

### LINKED ISSUE #22125: Required image format for MQTT camera not documented
### Feedback

from the documentation it's not clear what the topic expects. And image url? An image file? In which format? ...?

### URL

https://www.home-assistant.io/integrations/camera.mqtt/

### Version

2022.3.6

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Clarify MQTT camera documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #22125

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by emontnemery** [comment]
It's not documented here what image formats are supported by the camera entity https://developers.home-assistant.io/docs/core/entity/camera, should it be?

File: source/_integrations/camera.mqtt.markdown [hunk 1]
```diff
@@ -8,7 +8,7 @@ ha_iot_class: Configurable
 ha_domain: mqtt
 ---
 
-The `mqtt` camera platform allows you to integrate the content of an image file sent through MQTT into Home Assistant as a camera. Every time a message under the `topic` in the configuration is received, the image displayed in Home Assistant will also be updated.
+The `mqtt` camera platform allows you to integrate the content of an image file sent through MQTT into Home Assistant as a camera. Every time a message under the `topic` in the configuration is received, the image displayed in Home Assistant will also be updated. Messages received on `topic` should contain the full contents of an image file, for example a jpg image, without any additional encoding or metadata.
```
Comment (frenck) [inline source/_integrations/camera.mqtt.markdown]:
```suggestion
The `mqtt` camera platform allows you to integrate the content of an image file sent through MQTT into Home Assistant as a camera. Every time a message under the `topic` in the configuration is received, the image displayed in Home Assistant will also be updated. Messages received on `topic` should contain the full contents of an image file, for example, a JPEG image, without any additional encoding or metadata.
```

File: source/_integrations/camera.mqtt.markdown [hunk 2]
```diff
@@ -23,6 +23,11 @@ camera:
     topic: zanzito/shared_locations/my-device
 ```
 
+The sample configuration above can be tested by publishing an image to the topic from the console:
+```
```
Comment (frenck) [inline source/_integrations/camera.mqtt.markdown]:
````suggestion
The sample configuration above can be tested by publishing an image to the topic from the console:

```shell
````

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍

**Pull request merged**